consensus/consensus_algorithm.go

```diff
@@ -0,0 +1,69 @@
+package consensus
```

Comment:
Why consensus and not dolevstrong? What if we would want to have another consensus impl.?